hypothesis-python/src/hypothesis/stateful.py

```diff
@@ -185,9 +187,9 @@ def output(s):
                 # a return value using the variable name they are assigned to.
                 # See https://github.com/HypothesisWorks/hypothesis/issues/2341
                 if print_steps or TESTCASE_CALLBACKS:
-                    data_to_print = {
-                        k: machine._pretty_print(v) for k, v in data.items()
-                    }
+                    data_to_print = {}
+                    for k, v in data.items():
+                        data_to_print[k] = machine._pretty_print(v)
```

Comment:
Might as well undo this, for a smaller diff.  Each case isn't a big deal at all, but a habit of minimizing unrelated changes tends to make review (and if necessary future debugging) easier.